ocs_ci/templates/workloads/fsync.c

```diff
@@ -0,0 +1,24 @@
+#include <stdlib.h>
```

Comment:
fsync operation  can also be done through python, please check and see if .c file can be replaced with python file

Reply:
I doubt about it. As fsync here is system call.